plone-5.2.x.cfg

```diff
@@ -11,19 +11,5 @@ black = 20.8b1
 # Error: The requirement ('virtualenv>=20.0.35') is not allowed by your [versions] constraint (20.0.26)
 virtualenv = 20.0.35
 
-# Error: The requirement ('pep517>=0.9') is not allowed by your [versions] constraint (0.8.2)
-pep517 = 0.9.1
-
 # Error: The requirement ('importlib-metadata>=1') is not allowed by your [versions] constraint (0.23)
 importlib-metadata = 2.0.0
-
-# cryptography 3.4 requires a rust compiler installed on the system:
-# https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst#34---2021-02-07
-cryptography = 3.3.2
-
-# cffi 1.14.3 fails on apple m1
-cffi = 1.14.4
-
-# requirement for json widget tests to pass
-plone.schema = 1.3.0
-plone.dexterity = 2.9.8
```

Comment:
I found that issue #1018 occurs because of plone.dexterity 2.9.10+. More specifically after plone/plone.dexterity#137

Strangely, it only occurs in Python 2.

The error occurred in Jenkins, but it did not occur in plone.restapi GA, because the plone.restapi buildout was pinned to plone.dexterity 2.9.8.

After the merge of this PR, the error can happen in GA as well.

Reply:
I guess with newer dexterity more test failures could indeed happen on GH actions. If so, this is good: it should bring us closer to fixing the underlying problem, whether that is in 'real' code or test code.